wagtail/contrib/modeladmin/helpers/permission.py

```diff
@@ -162,4 +162,6 @@ def user_can_publish_obj(self, user, obj):
 
     def user_can_copy_obj(self, user, obj):
         parent_page = obj.get_parent()
+        if parent_page is None:
+            return True  # If there is no parent, assume they can copy it
```

Comment:
Meta: Is `True` correct here? It feels like it probably is, as making another root page won't really do anything unless it's also attached to a `Site`.

Reply:
Resolved with change of approach